src/VolleyManagement.Backend/VolleyManagement.UI/Areas/WebAPI/ViewModels/GameReports/StandingsEntryViewModel.cs

```diff
@@ -155,11 +150,13 @@ public static List<StandingsEntryViewModel> SetPositions(List<StandingsEntryView
         /// <param name="firstEntry">First team standings entry</param>
         /// <param name="secondEntry">Second team standings entry</param>
         /// <returns>True if teams have same position</returns>
+#pragma warning disable S1244 // Want to check floating point not with exact values, but with range
         public static bool EntriesHaveSamePosition(StandingsEntryViewModel firstEntry, StandingsEntryViewModel secondEntry)
         {
             return firstEntry.Points == secondEntry.Points
-                && firstEntry.SetsRatio == secondEntry.SetsRatio
-                && firstEntry.BallsRatio == secondEntry.BallsRatio;
+                && firstEntry.SetsRatio.Equals(secondEntry.SetsRatio)
```

Comment:
compare floats using 0.001 tolerance

Reply:
what it mean?

Reply:
tell me how properly compare floats/doubles? why they cannot be compared as integers?

Reply:
in integer we have precision 1, and in float/double we have 0,0000001 precision, ok I`ve read article on msdn 
![image](https://user-images.githubusercontent.com/17709640/37240798-ed1b8042-2458-11e8-8711-67ac7b026ae7.png)

Reply:
![image](https://user-images.githubusercontent.com/11724508/37241741-e2e52728-2466-11e8-8283-1afea20378f1.png)